CMakeLists.txt

```diff
@@ -29,10 +29,9 @@ cmake_minimum_required (VERSION 2.8)
 project(kindr)
 
 set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR})
-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}" ${CMAKE_MODULE_PATH})
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 
 find_package(Eigen3 REQUIRED)
-
 include_directories(include)
 include_directories(${EIGEN3_INCLUDE_DIR})
```

Comment:
Hm, this still seems wrong because `EIGEN3_INCLUDE_DIR` is nowhere defined. (See https://github.com/ethz-asl/kindr/blob/master/cmake/FindEigen.cmake). Why does that work for you? I'd recommend to remove the link `FindEigen3.cmake` and use it without `3` in this file. Or rename instead and fix this file instead : https://github.com/ethz-asl/kindr/blob/master/test/CMakeLists.txt.